### ISSUE TITLE
Use force:source:deploy

### ISSUE THREAD

**Issue by sivanandjothy** [azlamsalam/sfpowerscripts#128]
**Steps to Replicate**

1. Convert mdapi to source format using sfdx force:mdapi:convert
2. Step 1 resulted in 7282 components (29.3 MB size on disk including 8.87 MB of static resources)
3. Deploy using sfpowerscripts tasks "Deploy Source To the provided org using metadata api"

**Expected results**
Use force:source:deploy

**Actual Results**
Converts source to mdapi again and deploys.

**Comment by azlamsalam** [comment]
Using force:source: deploy at the moment makes it really hard to parse the deployment id, so investigating this
